src/index.js

```diff
@@ -22,4 +43,34 @@ function renderTasks() {
   });
 }
 
+function addTask(description) {
+  const task = {
+    description,
+    completed: false,
+    index: tasks.length + 1,
+  };
+
+  tasks.push(task);
+  saveTasks();
+  renderTasks();
+}
+
+const newTaskForm = document.querySelector('form');
+newTaskForm.addEventListener('submit', (event) => {
+  event.preventDefault();
+  const newTaskInput = newTaskForm.querySelector('.new-task-input');
+  addTask(newTaskInput.value);
+  newTaskInput.value = '';
+});
+
+const clearCompletedBtn = document.querySelector('.clear-completed-btn');
+clearCompletedBtn.addEventListener('click', () => {
+  tasks = tasks.filter((task) => !task.completed);
+  tasks.forEach((task, i) => {
```

Comment:
- [ ] As mentioned in the [requirements](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m3_add_remove.md#project-requirements) please use a separate file for adding the new functionalities.

> - Create a new JavaScript file for the new functionality.

Reply:
well resolved! ✔️